PowerSkills.sln

```diff
@@ -44,6 +44,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "_SampleData", "_SampleData"
 		SampleData\Invoice_5.pdf = SampleData\Invoice_5.pdf
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CustomEntitySearch", "Text\CustomEntitySearch\CustomEntitySearch.csproj", "{F0B21155-829F-4B26-853B-82ECAAB18D23}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{F5F3F598-2DCA-42EA-9797-D3A3E7CFA2CB}"
+EndProject
```

Comment:
remove this project

Reply:
I thought I fixed this. Tried to just clean again.

Reply:
its still here in iteration 11

---
In reply to: [307531580](https://github.com/Azure-Samples/azure-search-power-skills/pull/1#discussion_r307531580) [](ancestors = 307531580)